app/templates/gentelella/admin/event/wizard/wizard.html

```diff
@@ -110,6 +110,7 @@ <h1 style="font-weight: 100; margin-top: 0;" id="page-title">{{ self.title() }}<
 
     {# Additional js functionality #}
     <script src="{{ url_for('static', filename='admin/lib/moment/min/moment.min.js') }}"></script>
+    <script src="{{ url_for('static', filename='admin/lib/moment-timezone/builds/moment-timezone-with-data.min.js') }}"></script>
```

Comment:
See my prev. comment. We can also remove this dependency. Moment timezone is big